src/plugins/console/server/lib/proxy_request.ts

```diff
@@ -113,7 +113,10 @@ export const proxyRequest = ({
 
   const timeoutPromise = new Promise<any>((timeoutResolve, timeoutReject) => {
     setTimeout(() => {
-      if (!req.aborted && !req.socket) req.abort();
```

Comment:
This condition won't be met since `req.socket` is always truthy, therefore `req.abort` is never executed when a timeout happens